### PR TITLE
Manish chart mail

### PR DESCRIPTION
**Pull request by Manish-Srans**
Check My Code.. Scheduler is not implemented right now. Today Night I'll Implement.

### CODE REVIEW

File: src/main/java/com/srans/nestserver/model/Bed.java [hunk 1]
```diff
@@ -45,72 +45,64 @@
...
-		this.vacatedDate="";
-		this.bedNo=1;
-		
-		
+		this.id = 0L;
+		this.hostelId = 0L;
+		this.floorId = 0L;
+		this.roomId = 0L;
+		this.alloted = ' ';
+		this.position = "";
```
Comment (srans-admin) [inline src/main/java/com/srans/nestserver/model/Bed.java:55]: Why u changed N to empty here?

File: src/main/java/com/srans/nestserver/model/Bed.java [hunk 2]
```diff
@@ -119,6 +111,14 @@ public void setPosition(String position) {
 		this.position = position;
 	}
 
+	public Character getAlloted() {
+		return alloted;
```
Comment (srans-admin) [inline src/main/java/com/srans/nestserver/model/Bed.java:115]: Can you check Booking Bed functionality disturbed with this?

File: src/main/java/com/srans/nestserver/model/User.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+/**
+	 * Date :24-December-2019
+	 * @author ManishKumar
+	 * @version 1.0
+	 *
+	 */
+	@Component
+	public class User {
+
+		private String emailAddress;
```
Comment (srans-admin) [inline src/main/java/com/srans/nestserver/model/User.java]: Why is this bean, only for sending email, why do we need seperate bean?

File: src/main/java/com/srans/nestserver/service/MailService.java [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+	/**
+	 * This function is used to send mail without attachment.
+	 * 
+	 * @param user
+	 * @throws MailException
+	 */
+
+	public void sendEmail(User user) throws MailException {
+
```
Comment (srans-admin) [inline src/main/java/com/srans/nestserver/service/MailService.java]: Man! I hope just copy pasted code, check what is our requirement, accordingly code, u need to send email to the tenant after registration, it means send email with all the info of tenant with nice template and use as part of the body itself , no attachment required, in your code I don't see from TenantControler you have called this service to send email with logic of sending tenant information which includes name, phone, email, etc along with booked hostel, floor, room, bed information.

**Review by srans-admin — CHANGES_REQUESTED** [review]

**Comment by srans-admin** [comment]
I dont see any new updates from you yet

**Review by srans-admin — CHANGES_REQUESTED** [review]
You have not replied to my comments and you don't understand the requirement properly it looks.

**Comment by Manish-Srans** [comment]
Ram here : I dont see any improvements in this PR, hence closing this and gnererated in seperate PR, with genreic implementions on email

**Pull request closed without merging**